src/runner.go

```diff
@@ -88,25 +103,6 @@ func processSignedScript(incomingContent []byte) string {
 		return "Yaml couldn't be unmarshaled"
 	}
 
-	// Set env variables
-	getEnvVarName := func(key string) string {
-		return fmt.Sprintf("RHC_WORKER_%s", strings.ToUpper(key))
-	}
-	for key, value := range yamlContent.Vars.ContentVars {
-		prefixedKey := getEnvVarName(key)
-		err := os.Setenv(prefixedKey, value)
-		if err != nil {
-			log.Errorln(err)
-		} else {
-			log.Infoln("Successfully set env variable", prefixedKey, "=", value)
-		}
-	}
-	defer func() {
-		for key := range yamlContent.Vars.ContentVars {
-			os.Unsetenv(getEnvVarName(key))
-		}
-	}()
-
 	// NOTE: just debug to see the values
 	log.Debugln("Insights Signature:", yamlContent.Vars.InsightsSignature)
 	log.Debugln("Insights Signature Exclude:", yamlContent.Vars.InsightsSignatureExclude)
```

Comment:
We need to remember to remove those two logs later

Reply:
I just removed all of the debug logs now, I never used it, can be manually added anytime during local development setup anyway